### PR TITLE
fix: add codeowners file

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Comment by wtrocki** [comment]
Ok. I think since I do not have access to the repository I cannot be made owner. Please ignore then

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## All code changes
+* @MikeEdgar
+
+## API/SDKs
+/kafka-admin/.openapi @wtrocki
```
Comment (wtrocki) [inline .github/CODEOWNERS:5]: We might replace that with the team later on.

**Comment by wtrocki** [comment]
FYI @machi1990 I will be making this new team to simplify reviews that have OpenAPI files. Once team is created I will be looking to create codeowners in kas-fleetshard as well

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## All code changes
+* @MikeEdgar
+
+## API/SDKs
+/kafka-admin/.openapi @bf2fc6cc711aee1a0c2a/devexp-sdks
```
Comment (MikeEdgar) [inline .github/CODEOWNERS:5]: > Users must have read access to the repository and teams must have explicit write access, even if the team's members already have access

Looks like you need to be listed individually.

**Comment by wtrocki** [comment]
Ah.. it is write access then.. Ok. Bad idea. CLosing

**Comment by MikeEdgar** [comment]
@wtrocki it looks like _individuals_ need read access, but teams need write. I'm not sure why the difference. I think it's still achievable with the team members listed out.

**Comment by wtrocki** [comment]
It is still failing but I have write access. I would leave that for now considering that we might get new community for project sooner than later.

**Pull request closed without merging**